libunsf.c

```diff
@@ -3324,7 +3339,7 @@ UNSF_SYMBOL void unsf_convert_sf_to_gus(UnSF_Options *options) {
                                     if ((sf_num_instrument_indexes * 4 != subchunk.size) ||
                                         (sf_instrument_indexes)) BAD_SF();
 
-                                    sf_instrument_indexes = malloc(sizeof(sfInstBag) * sf_num_instrument_indexes);
+                                    sf_instrument_indexes = (sfInstBag *) malloc(sizeof(sfInstBag) * sf_num_instrument_indexes);
```

Comment:
> Which gcc?  All the ones I've tried haven't given a warning.
>

That was a response to your concern about casting when pointers
and integers are differently sized: gcc warns about that particular
one. And that case would not happen, because we do include
stdlib.h so gcc cannot default to int return for malloc(), and also we
know the type of the pointer we are allocating.